website/content/docs/discovery/checks.mdx

```diff
@@ -181,7 +77,7 @@ automatically monitor the health of a service instance or node.
   to temporarily remove one or all service instances on a node
   from service discovery DNS and HTTP API query results.
 
-### Script check ((#script-interval))
+### Script check
```

Comment:
The `((#script-interval))` is an anchor tag alias that preserves backwards compatibility for other content that links to information about this kind of health check. Do we have a need to break that backwards compatibility? If not, I recommend preserving the existing link aliases (for all check types, not just this one).

Reply:
CC @boruszak

Reply:
the reason is that in the original doc, the detailed section at the bottom with the examples and all, was a link to the redundant section at the top so for example http check pointed to http+interval. with the update and shortening of the top section it would be even more ridiculous

Reply:
I had previously restructured this page and was surprised to find the redundant info at the top. I'm not sure if I accidentally left that in, or if something weird happened in that merge or since. Either way, you're right that the bulleted items at the top were being linked to, not the details below, so we're effectively still preserving the old links with your change.

Cool - I'll dismiss my blocking review!